access/src/main/webapp/WEB-INF/jsp/fullRecord/neighborList.jsp

```diff
@@ -29,8 +29,18 @@
 			<c:set var="currentItemClass" scope="page">
 				<c:if test="${briefObject.id == neighbor.id}"> current_item</c:if>
 			</c:set>
+			<c:set var="markedForDeletion"  value="${cdr:contains(neighbor.status, 'Marked For Deletion')}" scope="request"/>
```

Comment:
i think you can use the default page scope for this, you don't need the variable past this point

Reply:
Changing the scope here causes the current record's icon to show for all records

Reply:
Changing it to `page` does? Weird, my understanding is `page` is for the current file, while `request` is for all pages in the request:
https://javapapers.com/jsp/explain-the-scope-of-jsp-objects/
It would likely be safest to change the name of the variable to something that doesn't overlap with the `markedForDeletion` variable from the controller, like `neighborMarkedForDeletion`.

Reply:
Seems like it needs the current setup so the current neighbor can override the controller set icon, changing the variable name results in the issue of all related items getting the same icon, even if they shouldn't. I'm guessing it needs to stay "request" so thumbnail.jsp gets the right value.

Reply:
Hmm, yeah i guess that makes sense since thumbnail.jsp is receiving this variable from the request rather than as a parameter passed to it explicitly, or a specially named variable just for the thumbnail component. Hopefully we won't have any issues where the state of the last neighbor changes content for the record the neighbor list is embedded in. Luckily its basically the last part of the page right now, so that reduces the chance of this issue. So we can go with this, and switch to passing in a parameter if we end up moving things around later.